### ISSUE TITLE
Improve chunk baking performance

### ISSUE THREAD

**Issue by Rosethorns** [RS485/LogisticsPipes#1338]
As nice as the bevelled pipes with support struts and so on are, logistics pipes existed with basic box rendering for the longest time.

I feel like until a solution can be found to the 1.7.10 - 1.12.2 render changes that logistics pipes should revert back to simple rendering for functionalities sake

**Comment by bziemons** [comment]
Sadly, the old render code is not ready for 1.12.2 and it would take time going backwards or forward. I prefer going forward tbh. In my personal experience the lag is very much bearable, even with many pipes in our base. But I understand that one can be annoyed.

**Comment by dblsaiko** [comment]
I could test my fast OBJ renderer (it uses VBOs and shaders and more fancy opengl stuff, instead of the fixed function pipeline, which is what I'm guessing is used now) with LP that I wrote to render a model with a lot of vertices without dropping the game down to 30fps, and see if that makes the lag any better. Not sure how it'll do with rendering a lot of smaller models instead of one big one though. We'll see

**Comment by bziemons** [comment]
Ah, forgot to ask what lag we are talking about. Is it the per frame render lag oder the lag until the chunk is rebuilt? Those have two different causes and I was assuming the later one, because the pipes don't cause any known lags in the frame to frame rendering.

**Comment by dblsaiko** [comment]
Oh right, pipe models are baked models. Disregard what I said above then

**Comment by Rosethorns** [comment]
Yeah, the lag caused when ever the chunk is rebuilt from adding/removing a pipe

**Comment by Bernasss12** [comment]
If I understood this correctly you are saying the game stutters everytime you add a pipe to your system right?
That's what theZorro266 was talking about, since the FPS lag (lag that lowers your fps in general) could indeed be caused by the more complex models (possibly) but the lag you are experiencing has (iirc) already been reported and is not necessarily related to the new pipe models.

**Comment by Bernasss12** [comment]
Could you see if adding or breaking blocks other than pipes very close to your LP system causes the same type of lag? Example attached:
[Video example](https://cdn.discordapp.com/attachments/484022895186542605/589824372739407878/CyCXnJYcKJ.mp4)

**Comment by Rosethorns** [comment]
the lag only occurs when logi pipes are added or removed from a chunk. Any close by models or blocks (for example IE multiblocks) do not cause the same stuttering when formed or broken.

**Comment by SonOfTheStars** [comment]
I did notuice this bug a while ago as well while I was playing Enigmatica 2: Expert. Adding or removing to my LP System was HECKA laggy. Causing a frame freeze for up to 2 seconds every single time.

Yet I am not having anywhere near the same problem right NOW using Omnifactory. 

Im thinking there might be a Optimization mod e.g. at work, that interferes with LP and causes these Spikes.

**Comment by Rosethorns** [comment]
I am not using any optimisation mods in the pack I am constructing
